src/app/checkout/order-success/order-success.component.html

```diff
@@ -1,67 +1,77 @@
 <div class="container" *ngIf="orderDetails">
-  <div class=col-12>
-    <h1>Thank you for your order!</h1>
+  <div class="col-12 thankyou shadow-sm mt-5 mb-3">
+    <h1>thank you for your order!</h1>
```

Comment:
spelling of `Thank you`

Reply:
Done, But it is handled using the CSS already